flow-editor/src/visual-node-editor/VisualNodeEditor.tsx

```diff
@@ -695,13 +692,16 @@ export const VisualNodeEditor: React.FC<VisualNodeEditorProps & { ref?: any }> =
             isLikelyTrackpad.current = true;
           }
 
+          const isTrackpad = e.deltaMode === 0; // Trackpad typically has deltaMode 0
+          const zoomSensitivity = isTrackpad ? 0.005 : 0.02; // Adjust sensitivity for trackpad vs mouse
+
           if (!isLikelyTrackpad.current) {
```

Comment:
You've added another variable determining whether it is a trackpad or not and did not change further conditions
The else branch never is applied if it is not detected as a trackpad